### ISSUE TITLE
Consider contributing ImageJ-OMERO details to the OMERO Guide Fiji repository

### ISSUE THREAD

**Issue by ctrueden** [imagej/imagej-omero#114]
See https://omero-guides.readthedocs.io/projects/fiji/ and https://github.com/ome/omero-guide-fiji/

**Comment by jburel** [comment]
@ctrueden good idea. We could add a section similar to what we did for [MoBIE and Zarr ](https://omero-guides.readthedocs.io/en/latest/fiji/docs/view_mobie_zarr.html) or [ImageJ Python](https://omero-guides.readthedocs.io/en/latest/fiji/docs/imagej_python.html)
